pom.xml

```diff
@@ -20,7 +20,7 @@ limitations under the License.
 
   <groupId>org.terracotta</groupId>
   <artifactId>offheap-store</artifactId>
-  <version>2.5-SNAPSHOT</version>
+  <version>2.5.0</version>
```

Comment:
This change shouldn't exist in the history of master. Master should move only from `2.4-SNAPSHOT` to `2.5-SNAPSHOT`. The move to `2.5.0` should happen in a dangling commit off the now 2.5-SNAPSHOT master line.

Reply:
This might explain better:
![Release Graph](https://user-images.githubusercontent.com/954331/69370730-85434100-0c6c-11ea-998e-ac6e032d355f.png)